TheMet/Model/TheMetStore.swift

```diff
@@ -53,18 +53,29 @@ class TheMetStore: ObservableObject {
 
   func fetchObjects(for queryTerm: String) async throws {
     if let objectIDs = try await service.getObjectIDs(from: queryTerm) {  // 1
-      for (index, objectID) in objectIDs.objectIDs.enumerated()  // 2
-      where index < maxIndex {
-        if let object = try await service.getObject(from: objectID) {
-          await MainActor.run {
-            objects.append(object)
-          }
+      for try await object in getResponseStream(for: objectIDs) {
+        await MainActor.run {
```

Comment:
Forgive me as I get more used to the new concurrency model, but is there a more elegant way to mutate a `@Published` array? I thought about marking it `@MainActor` but then it would have to be nonisolated on the Actor and that isn't allowed with the property wrapper.